Makefile

```diff
@@ -196,25 +195,11 @@ bundle: manifests setup kustomize ## Generate bundle manifests and metadata, the
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
 	./scripts/csv_description_update.sh update_csv
 
-#	$(KUSTOMIZE) build config/kustomize/crd -o internal/deploy/kustomize/daily/base/websphere-liberty-crd.yaml
 	$(KUSTOMIZE) build config/kustomize/crd -o deploy/kustomize/daily/base/runtime-component-crd.yaml
 	cd config/kustomize/operator && $(KUSTOMIZE) edit set namespace $(KUSTOMIZE_NAMESPACE)
-#	$(KUSTOMIZE) build config/kustomize/operator -o internal/deploy/kustomize/daily/base/websphere-liberty-deployment.yaml
 	$(KUSTOMIZE) build config/kustomize/operator -o deploy/kustomize/daily/base/runtime-component-operator.yaml
-# This does two replacements 
-# ${IMG} becomes ${KUSTOMIZE_IMG}
-# serviceAccountName: controller-manager becomes serviceAccountName: websphere-liberty-controller-manager
-# The second one looks unneeded, as the equivalent is already in the deployment file
-# TODO iain
-# Not sure about the first
-# Probably
-#	sed -i.bak "s,${IMG},${KUSTOMIZE_IMG},g;s,serviceAccountName: controller-manager,serviceAccountName: websphere-liberty-controller-manager,g" internal/deploy/kustomize/daily/base/websphere-liberty-deployment.yaml
-	sed -i.bak "s,${IMG},${KUSTOMIZE_IMG},g" deploy/kustomize/daily/base/runtime-component-operator.yaml
```

Comment:
@idlewis In order for this change to be made, the image value within kustomize/daily folder ([here](https://github.com/application-stacks/runtime-component-operator/blob/f2ef1e31f624a80ec7e7caaebebc51938e934c6a/deploy/kustomize/daily/base/runtime-component-operator.yaml#L293)) should have the new image